libs/ui/src/components/MediaItem/MediaItem.vue

```diff
@@ -138,11 +112,21 @@ export default {
     max-width: 18.75rem;
     text-align: center;
   }
-
-  .nsfw-content {
-    cursor: pointer;
-    bottom: 3.125rem;
-    position: absolute;
+}
+.nsfw-action {
+  cursor: pointer;
```

Comment:
don't think this is needed
it is already a `<button>` and you should use `<neoButton>`